site/introducing.md

```diff
@@ -55,7 +55,7 @@ Controlling Scope containers goes beyond simple lifecycle operations. Scope can
 
 Weave Scope can visualize your app within:
 
- * Docker Datacenter
+ * Docker Datacenters
```

Comment:
Why? Docker Datacenter is a product https://www.docker.com/products/docker-datacenter